dashboard/app/views/lti/v1/iframe.html.haml

```diff
@@ -0,0 +1,40 @@
+!!!
+%html{lang: "en"}
+  %head
+    %meta{charset: "UTF-8"}
+    %meta{name: "viewport", content: "width=device-width, initial-scale=1.0"}
+    %title= t('lti.iframe_title')
+    :javascript
+      window.onload = function() {
+        const auth_url = `#{@auth_url}`;
+        const iframe_message = `#{t('lti.iframe_message')}`;
+        const iframe_button = `#{t('lti.iframe_button')}`;
+        // Determine if we are running in an iframe
+        if (window.location !== window.parent.location) {
+          var messageDiv = document.createElement("div");
```

Comment:
I'm open to styling this differently, this currently gets the job done.

Reply:
Could we get someone from the design team to do this for us?

Reply:
I talked to Ali offline, he's created a jira ticket to follow up with UI enhancements. https://codedotorg.atlassian.net/browse/P20-769